packages/block/docs/classes/Block.md

```diff
@@ -52,12 +52,12 @@ Use the static factory methods to assist in creating a Block object from varying
 
 #### Parameters
 
-| Name | Type | Default value |
-| :------ | :------ | :------ |
-| `header?` | [`BlockHeader`](BlockHeader.md) | `undefined` |
-| `transactions` | `TypedTransaction`[] | `[]` |
-| `uncleHeaders` | [`BlockHeader`](BlockHeader.md)[] | `[]` |
-| `opts` | [`BlockOptions`](../interfaces/BlockOptions.md) | `{}` |
+| Name           | Type                                            | Default value |
+| :------------- | :---------------------------------------------- | :------------ |
+| `header?`      | [`BlockHeader`](BlockHeader.md)                 | `undefined`   |
+| `transactions` | `TypedTransaction`[]                            | `[]`          |
+| `uncleHeaders` | [`BlockHeader`](BlockHeader.md)[]               | `[]`          |
+| `opts`         | [`BlockOptions`](../interfaces/BlockOptions.md) | `{}`          |
```

Comment:
IS THIS AUTOMATICALLY FORMATTING MD TABLES??? THAT WOULD BE MY DREAMS COME TRUE!!! 🤩 😋

Reply:
Oh. 😬 😂 🙂

Reply:
I somehow always thought that Prettier would have been "merged" into ESLint respectively the functionality being included as a subfunctionality-cluster to be executed automatically.

So is there also a corresponding plugin for VSCode to have this done automatically on coding?